docs/source/crawler/basics.rst

```diff
@@ -20,12 +20,57 @@ a modified dictionary as its output once.
 >>> next(add1_node({'input': 1}))
 {'input': 2}
 
+.. Generators::
```

Comment:
This is an illegal/unkown paragraph markup. It causes the doc build to fail. Maybe this is just a ".. note::"?

Alternatively, you could put this into `docs/source/glossary.rst` and reference it via `:term:`generator` markup.